### PR TITLE
Completed projects page

### PR DESCRIPTION
**Pull request by Blackbullet99**

### CODE REVIEW

File: src/pages/AdminPage/views/Project/Project.jsx
Comment (Josh-Ay) [inline src/pages/AdminPage/views/Project/Project.jsx:1]: Remove the `setProjectsLoading`, its state is updated already in the context file.

The `projectsLoading` is also not required as what you need to track using the spinner is when the project time data has been fetched

Comment (Blackbullet99) [reply src/pages/AdminPage/views/Project/Project.jsx:1]: Issue has been resolved

Comment (Blackbullet99) [reply src/pages/AdminPage/views/Project/Project.jsx:1]: Search functionality complete and corrected projectTime loading state

Comment (Josh-Ay) [reply src/pages/AdminPage/views/Project/Project.jsx:1]: Reviewed the updates, nice work

File: src/pages/AdminPage/views/Project/ProjectEdit.jsx
[the remark this answers is not in the document]
Comment (Blackbullet99) [reply src/pages/AdminPage/views/Project/ProjectEdit.jsx:1]: All changes have been resolved

**Review by Josh-Ay — APPROVED** [review]

**Pull request merged**